### ISSUE TITLE
Error Occurred When Editing Spigot.yml

### ISSUE THREAD

**Issue by BluBoy3ch0** [Minehut/Meta#620]
### Bug report

**Checklist**
- [X] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [X] I am NOT reporting a security vulnerability
- [X] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [X] This issue does not contain any sensitive information

**Platform**

Website

**Describe the bug**

Updating spigot.yml just returns a Error occured, try again
Trying again does the same thing. 

**To reproduce**
Steps to reproduce the behaviour:
1. Go to 'the file manager'
2. Click on 'spigot.yml'
3. Save File

**Expected behaviour**

Expected, no error. Should save file. instead it doesnt save and giives an error

**Device Information**

- Minecraft client (and version): N/A
- Web browser (and version): Chrome 90.0.4430.93 (Official Build) (64-bit)
- Operating system (and version): Microsoft Windows 10 Home ; 10.0.19042 Build 19042
- Minehut server name: duprs

**Screenshots**

![image](https://user-images.githubusercontent.com/44213317/117526486-5745df80-af7a-11eb-9772-325cdbd77ced.png)

**Additional context**

I am simply just editing a value, changed something from 6000 to 1200, changed something from 1 to 3, I am not editing anything within the override.
---
**Try to reproduce this bug and let us know in the comments if you can reproduce it or not. Make sure to include your relevant device information!**

**Comment by awaffle1** [comment]
Minehut's file manager is generally bad and experiences issues all the time, it's not an issue with the file, it's just Minehut being weird, try again and it should work

**Comment by BluBoy3ch0** [comment]
Yeah I haven't touched it in a few days, i'll go back and see if it works
